src/status_im/contexts/onboarding/events.cljs

```diff
@@ -78,30 +78,28 @@
  (fn [_ [error]]
    {:dispatch [:biometric/show-message (ex-cause error)]}))
 
-(rf/defn create-account-and-login
```

Comment:
FYI: I simply transformed from the legacy `rf/defn` + `rf/merge` to `reg-event-fx` and `:fx` because then I could use the dispatched events as "hooks" to intercept and track.